.github/actions/spell-check/expect.txt

```diff
@@ -410,8 +361,6 @@ DENORMAL
 Deondre
 depersist
 deprioritized
-deref
-deserunt
 DESKTOPABSOLUTEEDITING
 DESKTOPABSOLUTEPARSING
 desktopshorcutinstalled
```

Comment:
not sure where it is thrown. But it has a typo: desktopshor**t**cutinstalled

Reply:
Yeah I saw a few other cases done time ago. I'll look over the entire list.